package.json

```diff
@@ -37,8 +37,7 @@
     "cover": "NODE_PATH=src NODE_ENV=test istanbul cover _mocha -- $npm_package_options_mocha test/unit/*.test.js test/unit/**/*.test.js",
     "test": "npm run test:unit && npm run test:integration",
     "test:unit": "NODE_PATH=src NODE_ENV=test mocha $npm_package_options_mocha test/unit/*.test.js test/unit/**/*.test.js",
-    "test:integration": "docker-compose run --rm worker /bin/sh -c \"npm run test:integration:raw\"",
-    "test:integration:raw": "NODE_ENV=test mocha $npm_package_options_mocha test/integration/*.test.js test/integration/**/*.test.js",
+    "test:integration": "NODE_ENV=test mocha $npm_package_options_mocha test/integration/*.test.js test/integration/**/*.test.js",
```

Comment:
would still be great to easily run tests locally.

Reply:
I added a new `test:integration:docker` command